packages/next/cms/components/ProfileRenderer.js

```diff
@@ -44,6 +44,8 @@ function ProfileRenderer({
   customSections = {},
   relatedTile,
   t,
+  opened,
```

Comment:
`searchOpened`/`searchHandlers`?

Reply:
Updated props naming. to searchOpened & searchHandlers!